plugin/trino-snowflake/src/main/java/io/trino/plugin/snowflake/ImplementAvgBigint.java

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
This commit doesn't contain any tests.

Reply:
There are tests cover the `avg` pushdown under connector behavior `SUPPORTS_AGGREGATION_PUSHDOWN`.
could let me know where I should add tests, thanks

Reply:
It means the 1st commit doesn't pass CI. Note that every commit should pass CI. 
I squashed commits into one and updated the commit message.